pkg/config/remote/meta/meta.go

```diff
@@ -53,6 +54,21 @@ func RootsConfig() EmbeddedRoots {
 	return rootsConfig
 }
 
+// RootsConfigUser returns all the roots of the user config repo
+func RootsConfigUser() (EmbeddedRoots, error) {
+	if configRoot := config.Datadog.GetString("remote_configuration.unstable.self_signed_root"); configRoot != "" {
+		return EmbeddedRoots{
+			1: EmbeddedRoot(configRoot),
+		}, nil
+	}
+	return nil, fmt.Errorf("missing root for the user self-signed remote-configuration repository")
+}
+
+// First returns the first root the EmbeddedRoots
+func (roots EmbeddedRoots) First() EmbeddedRoot {
```

Comment:
There is a risk that people using type EmbeddedRoots do not notice this method and use `roots[0]` instead. It would be safer to have EmbeddedRoots a struct with an non-exported mapping and that can only be used through "safe" methods like this one.

Reply:
Will add it in followup RCM-115